view/rooms/new/app.js

```diff
@@ -54,6 +55,28 @@ function getPathVariables(adr){
   };
 }
 
+function createRoom(paramaters){
+  const req = request.post('/v1/rooms');
+  const obj = {
+    name: paramaters.name,
+    type: paramaters.type,
```

Comment:
`roomType` instead of `type` is used previously in this file. Autoredirect doesn't work when specifying `type`

Reply:
replaced type with roomType